docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -444,20 +446,16 @@ Other attributes that are available to {walkthrough-name} authors are:
 
 * Default attributes:
 ** OpenShift App Host: `\{openshift-app-host}`
-** Che URL: `\{che-url}`.
+** CodeReady Workspaces URL: `\{che-url}`.
 ** Fuse URL: `\{fuse-url}`
 ** Launcher URL: `\{launcher-url}`
 ** API Management URL: `\{api-management-url}`
 ** AMQ URL: `\{amq-url}`
 ** AMQ Broker URL: `\{amq-broker-tcp-url}`
 ** AMQ Credential Username: `\{amq-credentials-username}`
 ** AMQ Credential Password: `\{amq-credentials-password}`
-** EnMasse URL: `\{enmasse-url}`
-** EnMasse Broker URL: `\{enmasse-broker-url}`
-** EnMasse Credential Username: `\{enmasse-credentials-username}`
-** EnMasse Credential Password: `\{enmasse-credentials-password}`
```

Comment:
Is this correct? We still have EnMasse in the form of AMQ Online, I'm not aware whether we've updated the variable names to not use `enmasse-*`. Have we updated these variable names?

We no longer have AMQ though. I'd imagine `amq-url`, `amq-broker-tcp-url`, `amq-credentials-username` and `amq-credentials-password` to no longer exist.

Reply:
oops, thanks for the catch, had fat fingered copying from walkthrough.